src/coreclr/nativeaot/System.Private.CoreLib/src/System/GC.NativeAot.cs

```diff
@@ -273,6 +273,42 @@ public static bool TryStartNoGCRegion(long totalSize)
             return StartNoGCRegionWorker(totalSize, false, 0, false);
         }
 
+        internal static Action? _NoGCCallback;
```

Comment:
This should have `s_` prefix and start with lowercase to follow https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md